.github/workflows/ci.yaml

```diff
@@ -6,9 +6,9 @@ on:
 name: ci
 env:
   GOOGLE_CLOUD_PROJECT: "span-cloud-testing"
-  GOOGLE_CLOUD_INSTANCE: "spanner-testing"
+  GOOGLE_CLOUD_INSTANCE: "pgadapter-testing"
```

Comment:
The `spanner-testing` instance is in us-central1. That region is no longer allow-listed as a region where PG databases may be created, and will cause an `INVALID_ARGUMENT` error to be returned when we try to create a database. `pgadapter-testing` is located in us-east1 which is allowed to create PG databases.

Reply:
Shouldn't it be in nam7-preview?

I believe the list of pg compatible allocs is
nam7-preview, europe-north1, us-west2, asia-east2

Reply:
According to the UI these are the allowed regions:

![image](https://user-images.githubusercontent.com/1196707/152743796-919675c4-9154-45c4-847d-fbbe75bbfe43.png)